src/Adapter/Order/CommandHandler/AddProductToOrderHandler.php

```diff
@@ -349,6 +396,59 @@ private function createSpecificPriceIfNeeded(
         return null;
     }
 
+    /**
+     * @param Cart $cart
```

Comment:
Add comment, something like: `This function extracts the newly added product from the cart and reformat the data in order to create a dedicated OrderDetail with the appropriate amounts`
And maybe rename into `getNewCartProduct` or `getNewCartProductData`

Reply:
Fixed